### PR TITLE
update mopub interstitial adapter and remove admob banner adapters

### PR DESCRIPTION
**Pull request by applovinjosh**

### CODE REVIEW

File: MoPub/Android/AppLovinInterstitialAdapter.java [hunk 1]
```diff
@@ -22,9 +23,11 @@
 {
     private CustomEventInterstitial.CustomEventInterstitialListener mInterstitialListener;
     private Activity                                                parentActivity;
-    private AppLovinAdService                                       adService;
+    private AppLovinSdk                                             sdk;
     private AppLovinAd                                              lastReceived;
 
+    private static final String TAG = "AppLovinAdapter";
```
Comment (thomasmso) [inline MoPub/Android/AppLovinInterstitialAdapter.java]: Shall we should add some context to the `TAG` to indicate this is coming from a MoPub adapter specifically?

File: MoPub/Android/AppLovinInterstitialAdapter.java [hunk 2]
```diff
@@ -43,11 +46,11 @@ public void loadInterstitial(Context context, CustomEventInterstitial.CustomEven
             return;
         }
 
+        Log.d( TAG, "Request received for new interstitial." );
```
Comment (thomasmso) [inline MoPub/Android/AppLovinInterstitialAdapter.java]: Same comment as above

File: MoPub/Android/AppLovinInterstitialAdapter.java [hunk 3]
```diff
@@ -100,7 +103,7 @@ public void onInvalidate()
     @Override
     public void adReceived(AppLovinAd ad)
     {
-        Log.d( "MoPub", "AppLovin interstitial loaded successfully." );
+        Log.d( TAG, "AppLovin interstitial loaded successfully." );
```
Comment (thomasmso) [inline MoPub/Android/AppLovinInterstitialAdapter.java]: Same comment as above

File: MoPub/Android/AppLovinInterstitialAdapter.java [hunk 4]
```diff
@@ -57,22 +60,22 @@ public void showInterstitial()
 
         if ( adToRender != null )
         {
-            Log.d( "MoPub", "Showing AppLovin interstitial ad..." );
+            Log.d( TAG, "Showing AppLovin interstitial ad..." );
```
Comment (thomasmso) [inline MoPub/Android/AppLovinInterstitialAdapter.java]: Same comment as above

**Comment by thomasmso** [comment]
Looks good.

**Comment by leolobato** [comment]
I couldn't find any reference/documentation as to why `AppLovinCustomEventBanner` has been removed from the repository except for this commit.

Is it documented anywhere on AppLovin website? Do they not support Admob banners mediation anymore?

**Pull request merged**